.github/actions/deploy/action.yml

```diff
@@ -5,9 +5,9 @@ inputs:
   artifact-name:
     required: true
     description: "Name of the artifact to deploy"
-  nuget-api-key:
+  nuget-user:
```

Comment:
Is reading from `${{ vars.NUGET_USER}}` an option? I'm mostly curious. I don't mind this as an input.

Reply:
I don't think so. Custom actions can't access secrets and variables directly.